Views/ShoppingCart/Index.cshtml

```diff
@@ -0,0 +1,88 @@
+@model ShoppingCartViewModel
+@using OrchardCore.Commerce.Abstractions
+@using OrchardCore.Commerce.Money
+<zone name="Header">
+    <header class="masthead">
+        <div class="container">
+            <div class="intro-text">
+                <div class="intro-lead-in"></div>
+                <div class="intro-heading text-uppercase">@T["Your cart"]</div>
+            </div>
+        </div>
+    </header>
+</zone>
+<section id="shopping-cart">
+    @if (Model.Lines != null && Model.Lines.Any())
+    {
+    <table class="table">
+        <thead><tr><th class="col-1">@T["Quantity"]</th><th class="col-9">@T["Product"]</th><th class="col-1">@T["Price"]</th><th class="col-1">@T["Action"]</th></tr></thead>
```

Comment:
`col-` css classes do not exist in Bootstrap 4.3.1 for `<th>`.

In order to set the width of each column, you need to use inline style or additional css classes that will do the equivalent, like this basic sample:

```
<th style="width:10%">Quantity</th>
<th style="width:70%">Product</th>
<th style="width:10%">Price</th>
<th style="width:10%">Action</th>
```

Reply:
Don't want to detract too much,  but is there any chance the checkout pave can move away from using a table architecture? I don't know what the alternative is, I just know in Orchard I had to make all the tables scroll left to right otherwise mobile users had a difficult time checking out.

Reply:
Mini cart summary could work. Look at how Shopify does it.

Reply:
Good distinction to make 😉

Reply:
@jeffolmstead I think I'll give it a try tonight. <rant>It's slightly depressing that HTML went from unsemantic tables for everything to divs + CSS to semantic markup, and back to unsemantic divs with semantics defined in CSS.</rant> :D

Reply:
You could try Flexboxes.

Reply:
I ripped tables out of the cart.